### ISSUE TITLE
Zombies are looking up 1.20

### ISSUE THREAD

**Issue by SemKa1407SK** [ralian/eai#36]
Zombies look up and shoot up as well, how to fix?

**Comment by deannreid** [comment]
This was replaced with DayZ Expansion AI

**Comment by SemKa1407SK** [comment]
Дейзи Экспансион не установлен

**Comment by LOWZONEDAYZ** [comment]
This is not entirely true
The EnfusionAI is continued under Hunterz and his eAI_Reloaded mod
https://steamcommunity.com/sharedfiles/filedetails/?id=2905606565&searchtext=ai
The mod is still work in progress but has made already some huge steps
Its working and many Server uses this continued mod already
You dont need to install and use Expansion at all

**Comment by salutesh** [comment]
> This is not entirely true The EnfusionAI is continued under Hunterz and his eAI_Reloaded mod https://steamcommunity.com/sharedfiles/filedetails/?id=2905606565&searchtext=ai The mod is still work in progress but has made already some huge steps Its working and many Server uses this continued mod already You dont need to install and use Expansion at all

Yes this is also true but then you should report this in his discord and not in this repository where he has no control over or might never reads your message.
